src/praisonai-agents/tests/unit/context/test_conversation_compaction.py

```
@@ -0,0 +1,277 @@
+        compactor = IntelligentConversationCompactor(
+            analyzer=analyzer,
+            min_compaction_ratio=0.5  # Require 50% compression
+        )
+        
+        # Small conversation that won't achieve 50% compression
+        messages = [
+            {"role": "user", "content": "Short question"},
+            {"role": "assistant", "content": "Short answer"},
+        ]
+        
+        result, context = compactor.compact_conversation(
```

Ruff (0.15.15)

[warning] Unpacked variable `context` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)